### PR TITLE
Prepare CSU for initial public release

### LINKED ISSUE #461: Fix the text between $ sign turned into math expression
## Description

Somehow need to fix the sentence ("20 for the product...") in between two $ signs that have been turned into a math expression. Needs to be plain text. In http://cs-unplugged-develop.appspot.com/en-gb/topics/error-detection-and-correction/unit-plan/#real-world-implications

image

### LINKED ISSUE #478: Update CS Unplugged site text
Update text on homepage:

> CS Unplugged is a collection of free teaching material that teaches Computer Science through engaging games and puzzles that use cards, string, crayons and lots of running around. The resources available to teachers include unit plans, lesson plans, teaching videos, curriculum integration activities and programming exercises to plug in the computer science concepts they have just learnt unplugged.

### LINKED ISSUE #480: Store version and changelog
The documentation should store the changelog for the system, and the website should display the version number with a link to the changelog.

### LINKED ISSUE #481: Add alpha notice to website

### LINKED ISSUE #482: Improve homepage layout
- [x] Compress banner image.
- [x] Replace button icons (test displaying off button).
- [x] Shrink width of lead text.

### PR DESCRIPTION
**Pull request by JackMorganNZ**
- Store version and changelog (fixes #480).
  - Note: The changelog is already out of date, will be completed in the release branch.
- Add alpha notice to website (fixes #481).
- Update homepage text (fixes #478).
  - Keen for feedback on new design (I'm not 100% on it but seems to be an improvement).
- Fix the text between $ sign turned into math expression (fixes #461).
- Compress homepage banner image - 85% reduction! (fixes #482).

### CODE REVIEW

File: csunplugged/config/context_processors/version_number.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+"""Context processor for displaying version number."""
+
+from config import __version__
+
+
+def version_number(request):
+    """Return a string of the version number.
```
Comment (ravenmaster001) [inline csunplugged/config/context_processors/version_number.py]: Return a string, oh but Returns: A dictionary????

File: docs/source/changelog.rst [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+This page lists updates to CS Unplugged.
+All notable changes to this project will be documented in this file.
+
+We base our numbering system from the guidelines at `Semantic Versioning 2.0.0`_,
+however since our project started before it was migrated to GitHub, the first
+major open source release is being labeled as ``2.0.0``.
+
+Given a version number MAJOR.MINOR.HOTFIX:
+
+- MAJOR version change when major text modifications are made (for example: new
```
Comment (hayleyavw) [inline docs/source/changelog.rst]: what if major backend changes are made?

File: docs/source/changelog.rst [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  - Seven curriculum integrations.
+
+- Error detetction and correction:
+
+  - Two lessons for ages 8 to 11.
+  - Five curriculum integrations.
+
+- Sorting networks:
+
+  - One lesson for ages 8 to 10.
```
Comment (hayleyavw) [inline docs/source/changelog.rst]: no mention of unit plans or programming exercises anywhere?

File: docs/source/changelog.rst [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
```
Comment (hayleyavw) [inline docs/source/changelog.rst]: I think the changelog needs to include backend changes - particularly as devs are far more likely to read this than authors.

**Comment by hayleyavw** [comment]
Notes from visual inspection of "about" pages:
- [ ]  "computer science" should be capitalised on the home page 
  > and programming exercises to plug in the computer science concepts they have just learnt unplugged
- [ ] Sarang's job needs to be updated 
  > Sarang Love Leehan - Founder & Writer
- [ ] J-Dawg should be added to the team (might not have written any code, but he's been good for ideas here and there)
-  [ ] Maybe "current team" isn't quite accurate, should just be "version 2 contributors" (or something of the like)
- [ ] @ravenmaster001 and I think you should be shuffled up the list and "Technical Team Lead" - no arguing, give yourself more credit
- [ ] We should use our department email address, rather than just Tim's.
- [ ] On the principles page, capitalisation of computer science should be fixed
  > projects for promoting computer science

**Comment by hayleyavw** [comment]
Also just noticed this on my branch and the dev site.
screenshot from 2017-06-20 11-38-25

Spotted on the resource pages, no ages are displayed.

**Comment by JackMorganNZ** [comment]
This is ready for re-review!

File: csunplugged/templates/general/people.html [hunk 5]
```diff
@@ -40,10 +40,11 @@
     <li>Caitlin Duncan - Content Writer</li>
     <li>Tracy Henderson - Content Writer</li>
     <li>Behshid Ghorbani - Content Writer</li>
-    <li>Sarang Love Leehan - Founder &amp; Writer</li>
+    <li>Sarang Love Leehan - Creative Content and Illustration</li>
```
Comment (hayleyavw) [inline csunplugged/templates/general/people.html]: > Creative Content Developer and Illustrator

File: csunplugged/templates/general/people.html [hunk 6]
```diff
@@ -40,10 +40,11 @@
...
+    <li>Jack Morgan - Technical Team Lead</li>
```
Comment (ravenmaster001) [inline csunplugged/templates/general/people.html]: Office Dad?
:disappointed_jack:

**Pull request merged**